src/pages/[...slug].tsx

```diff
@@ -1,76 +1,70 @@
-import { parseSearchState, SearchProvider, useSession } from '@faststore/sdk'
```

Comment:
Noticed when clicking the nav links that the products here weren't getting updated. A full page refresh was required to update it.

https://user-images.githubusercontent.com/381395/164520813-0f4f8aaa-fa35-49f4-af3c-29858a4a12a4.mp4

Reply:
nice catch! fixed!